sunpy/image/coalignment.py

```diff
@@ -64,9 +66,8 @@ def calculate_shift(this_layer, template):
         Pixel shifts ``(yshift, xshift)`` relative to the offset of the template
         to the input array.
     """
-    # Repair any NANs, Infs, etc in the layer and the template
```

Comment:
Per the deprecation policy, we need to preserve this as the default behavior for now. See my comment on the main thread for one possible solution.